### PR TITLE
Add X

### PR DESCRIPTION
**Pull request by GingertronMk1**

### CODE REVIEW

File: _shows/18_19/a_girl_is_a_half_formed_thing.md [hunk 1]
```diff
@@ -73,7 +73,15 @@ links:
     author: Georgia Butcher
     title: "Behind the Scenes at NNT: A Girl Is A Half-Formed Thing"
     date: 2018-11-02
-
+  - type: Review
+    href: https://www.hercampus.com/school/nottingham/theatre-review-nnt-s-girl-half-formed-thing?fbclid=IwAR2yVybME2MYUydxzQhNV7Nyy0w7mvgrquN6uackkOoLEUC4CZWZAjKo_NI
```
Comment (samozzy) [inline _shows/18_19/a_girl_is_a_half_formed_thing.md]: Can the bits after the ? go away?

Comment (GingertronMk1) [reply _shows/18_19/a_girl_is_a_half_formed_thing.md]: Yes, yes they can

**Comment by BenWoodford76** [comment]
@GingertronMk1 you happy for this to merge?

**Comment by GingertronMk1** [comment]
@BenWoodford76 I'm inclined to say wait for the Kev review first

**Pull request merged**